.github/workflows/linux.yml

```diff
@@ -62,6 +63,10 @@ jobs:
               -D DEAL_II_WITH_TRILINOS="ON" \
               -D DEAL_II_WITH_PETSC="ON" \
               -D DEAL_II_WITH_METIS="ON" \
+              -D DEAL_II_WITH_HDF5="ON" \
+              -D HDF5_INCLUDE_DIR="/usr/include/hdf5/openmpi/" \
+              -D HDF5_LIBRARY="/usr/lib/x86_64-linux-gnu/hdf5/openmpi/libhdf5.so" \
+              -D HDF5_HL_LIBRARY="/usr/lib/x86_64-linux-gnu/hdf5/openmpi/libhdf5_hl.so" \
```

Comment:
It should suffice to simply specify something like `CMAKE_PREFIX_PATH="/usr/lib/x86_64-linux-gnu/hdf5/openmpi;/usr/include/hdf5/openmpi"`